src/extractor/intersection/coordinate_extractor.cpp

```diff
@@ -185,7 +185,7 @@ util::Coordinate CoordinateExtractor::ExtractRepresentativeCoordinate(
      */
     const bool first_coordinate_is_far_away = [&first_distance, considered_lanes]() {
         const auto required_distance =
-            considered_lanes * 0.5 * ASSUMED_LANE_WIDTH + LOOKAHEAD_DISTANCE_WITHOUT_LANES;
```

Comment:
Any background why we removed the magic parameter here?

Reply:
i don't know exactly where `0.5` comes from, but assume that it is needed to reduce number of `considered_lanes` by factor 2 due to double counting.